package.json

```diff
@@ -64,7 +63,11 @@
     "flow-bin": "^0.42.0",
     "husky": "^0.13.3",
     "jest": "^19.0.2",
-    "lint-staged": "^3.4.0"
+    "lint-staged": "^3.4.0",
+    "babel-preset-react-native": "^2.0.0"
+  },
+  "peerDependencies": {
+    "babel-preset-react-native": "^2.0.0"
```

Comment:
Yes, we don't want to bump the preset version each time it changes manually, when you `react-native init` you will have it installed so why would the haul need to install it itself? Besides `preset-react-native` is tied to RN version, so we would want to use the one provided with RN, otherwise we will have constantly getting issues that sth doesn't work and it's probably because of outdated RN preset.